packages/neos-ui-editors/src/EditorEnvelope/index.js

```diff
@@ -54,15 +60,22 @@ export default class EditorEnvelope extends PureComponent {
         if (editorDefinition && editorDefinition.component) {
             const EditorComponent = editorDefinition && editorDefinition.component;
 
+            const restProps = omit(this.props, ['validationErrors', 'highlight']);
+            const {highlight, validationErrors} = this.props;
+            const isInvalid = validationErrors && validationErrors.length > 1;
```

Comment:
Shouldn't this be `> 0`? - as soon as one error occurs it is invalid I think

Reply:
fixed